.github/workflows/tox.yml

```diff
@@ -190,9 +195,14 @@ jobs:
         run: python -m pip install --upgrade tox ${{ matrix.toxdeps }}
         shell: sh
 
-      - run: python -m tox -e ${{ matrix.toxenv }} ${{ matrix.toxargs }} -- ${{ matrix.pytest_flag }} ${{ matrix.posargs }}
+      - run: python -m tox -e ${{ matrix.toxenv }} ${{ matrix.toxargs }} -- ${{ matrix.pytest_flag }} ${{ matrix.posargs }} ${{ fromJSON('["--junitxml results.xml", ""]')[matrix.pytest_results_summary == 'true' && matrix.pytest == 'true'] }}
```

Comment:
@ConorMacBride is there a mechanism to run `--junitxml results.xml` and the subsequent action only when `tox` is running `pytest`? Or would that need to be included in the `toxenv`?

Reply:
Yeah, that should be possible. We do that for the coverage report here:
https://github.com/OpenAstronomy/github-actions-workflows/blob/a7918d02a33a34288dd0182170b39c74a71ba81e/tools/tox_matrix.py#L135-L140
The `pytest_results_summary` input will need to be integrated into `tools/tox_matrix.py` for it to be included in the job matrix. Will need to be included in the "string parameter" dictionaries and also included as a `@click.option` decorator.

Reply:
Great! Should I manually serialize the script into the workflow after making the change?

Reply:
If you have pre-commit installed, you can just do `pre-commit run --all-files` and it'll update the serialization for you.